app/global.css

```diff
@@ -30,6 +30,7 @@
 
 html {
   min-width: 360px;
+  color-scheme: dark;
```

Comment:
Should this be in the block above for `prefers-color-scheme`? How does it work in light mode?